code/modules/hydroponics/grown.dm

```diff
@@ -89,6 +89,8 @@
 /obj/item/reagent_containers/food/snacks/grown/attack_self(mob/user)
 	if(seed && seed.get_gene(/datum/plant_gene/trait/squash))
 		squash(user)
+	if(seed.get_gene(/datum/plant_gene/trait/noreact))
+		user.visible_message("<span class='warning'>[user] shakes [src] vigorously!</span>", "<span class='userdanger'>You shake [src] vigorously!</span>")

```

Comment:
Why?

Reply:
this makes it so you can use it in-hand, instead of just throwing it at a mob to activate it- this is why the timer was so long